TypeCobol/Compiler/Sql/Model/SelectClause.cs

```diff
@@ -60,6 +61,58 @@ public SelectClause(SyntaxProperty<SelectionModifier> selectionModifier, IEnumer
             _selections = selections?.ToArray() ?? Array.Empty<Selection>();
         }
 
+        private static void DumpSelection(TextWriter output, Selection selection)
+        {   
+            ///TODO
+            switch (selection.Type)
+            {
+                case SelectionType.DotStar:
+                    output.Write("DotStarSelection");
+                    break;
+                case SelectionType.Expression:
+                    output.Write("ExpressionSelection");
+                    break;
+                case SelectionType.Star:
+                    output.Write("StarSelection");
+                    break;
+
+            }
+        }
+        public override void Dump(TextWriter output, int indentLevel)
+        {
+            string indent = new string(' ', 2 * indentLevel);
+            output.Write(indent);
+            output.WriteLine("SelectClause");
+            indentLevel++;
+            if (this.SelectionModifier != null)
+            {
+                output.Write(new string(' ',2* indentLevel));
+                output.WriteLine("- SelectionModifier=<" + this.SelectionModifier + ">");
+            }
+            if (this.Selections != null)
+            {
+                output.Write(new string(' ', 2 * indentLevel));
+                indentLevel++;
+                output.WriteLine("- Selections =[");
+                int i = 1;
+                foreach (var selection in this.Selections)
+                {
+                    output.Write(new string(' ', 2 * (indentLevel+1)));
+                    DumpSelection(output,selection);
+                    if (Selections.Last() == selection)
```

Comment:
Use a regular `for` instead and test the value of loop index.
You can also use `string.Join` with `',' + Environment.NewLine' as separator.